binding.gyp

```diff
@@ -19,12 +19,22 @@
     "configurations": {
       "Release": {
         "target_conditions": [
-          ["OS != 'win'", {
+          ['OS not in "ios mac win"', {
             "cflags+": ["-fdata-sections", "-ffunction-sections", "-flto", "-fvisibility=hidden"],
-            "ldflags+": ["-Wl,--gc-sections"]
+            "ldflags+": ["-Wl,--gc-sections"],
+            "defines+": ["_FORTIFY_SOURCE=2", "NDEBUG"],
+          }],
+          # Avoid defining _FORTIFY_SOURCE on Darwin
```

Comment:
Isn't this defining `_FORTIFY_SOURCE=2` twice for Linux? All three conditions will match.